### PR TITLE
fix(ui): various 

### PR DESCRIPTION
**Pull request by MiguelMedeiros**
This PR fixes:
- [x] [fix(wallet): change send navigation action](https://github.com/synonymdev/bitkit/commit/d09703565da99d6c77b09bc84a50c053c3d4b828)
- [x] [fix(onboarding): change copy text](https://github.com/synonymdev/bitkit/commit/d8fae33208408ecbd898379d40ea896ce6fd4aa1)
- [x] [fix(receive): change copy](https://github.com/synonymdev/bitkit/commit/eee071d93cbdcbccf295ee5e2af869a9d43be030)
- [x] [fix(wallet): change amount button size](https://github.com/synonymdev/bitkit/commit/ddc4a7615ca366b53fef93cb1848752c33219d85)
- [x] [fix(send): add bottomsheetbackpress to go back home](https://github.com/synonymdev/bitkit/pull/333/commits/37babc92e3c8f7fc02e89342090cdf3a30dd15da)
- [x] [fix(settings): backup wallet copy](https://github.com/synonymdev/bitkit/pull/333/commits/59d681eea31838dd3e680c209e2cb40e47fda770)
- [x] [fix: change carousel card background to black](https://github.com/synonymdev/bitkit/pull/333/commits/6fb6c313cf26e244f7d0e7a9dd1f561252d077bb)

### CODE REVIEW

File: src/navigation/tabs/TabNavigator.tsx [hunk 1]
```diff
@@ -88,27 +88,15 @@ export const TabBar = ({ navigation, state }): ReactElement => {
...
-					assetName: params.assetType,
-				},
-			});
-		} else {
-			toggleView({
-				view: 'sendNavigation',
-				data: {
-					isOpen: true,
-					snapPoint: 0,
-					initial: 'SendAssetPickerList',
```
Comment (pwltr) [inline src/navigation/tabs/TabNavigator.tsx:107]: `SendAssetPickerList` file and imports can be removed now as it is unused

File: src/screens/Wallets/AmountButtonRow.tsx [hunk 2]
```diff
@@ -94,10 +94,11 @@ const styles = StyleSheet.create({
 		justifyContent: 'space-between',
 		paddingVertical: 5,
 		paddingHorizontal: 5,
+		paddingTop: 15,
 	},
 	topRowButtons: {
-		paddingVertical: 5,
-		paddingHorizontal: 8,
+		paddingVertical: 10,
+		paddingHorizontal: 36,
```
Comment (pwltr) [inline src/screens/Wallets/AmountButtonRow.tsx]: padding here is too large now

Simulator Screen Shot - iPhone 13 - 2022-09-07 at 10 54 24

**Comment by pwltr** [comment]
Snap point. of Send BottomSheet should be larger, same as Receive (see `ReceiveNavigation.ts`)

Simulator Screen Shot - iPhone 13 - 2022-09-07 at 10 55 28

**Review by pwltr — CHANGES_REQUESTED** [review]

**Comment by pwltr** [comment]
Can you mark in the Google Doc which tasks you have done? Maybe strikethrough or something like that so we know what is still open?

File: src/screens/Onboarding/Slideshow.tsx [hunk 3]
```diff
@@ -142,8 +143,8 @@ const Slideshow = ({ navigation, route }: Props): ReactElement => {
 								<Display color="blue"> just a Tap.</Display>
 							</Display>
 							<Text01S color="gray1" style={styles.text}>
-								Experience the web without passwords. Use Slashtags to control
-								your profile, contacts & accounts.
+								Experience the web without passwords. Use Bitkit to control
+								profile, contacts & accounts.
```
Comment (pwltr) [inline src/screens/Onboarding/Slideshow.tsx]: Copy here changed again in Design v24 to:

"Experience the web without limits: portable profiles & feeds, dynamic contacts, passwordless accounts."

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/screens/Onboarding/Slideshow.tsx (hunk 3 above)
Comment (MiguelMedeiros) [reply src/screens/Onboarding/Slideshow.tsx]: Fixed here: [2d6646c](https://github.com/synonymdev/bitkit/pull/333/commits/2d6646c62c87dc9d3da915d7e26f6f08bd0c7af5)

File: src/navigation/tabs/TabNavigator.tsx (hunk 1 above)
Comment (MiguelMedeiros) [reply src/navigation/tabs/TabNavigator.tsx:107]: Fixed here: [9bf74a0](https://github.com/synonymdev/bitkit/pull/333/commits/9bf74a020f4b371f114553e49b287d1efc254bf0)

File: src/screens/Wallets/AmountButtonRow.tsx (hunk 2 above)
Comment (MiguelMedeiros) [reply src/screens/Wallets/AmountButtonRow.tsx]: Fixed here: [4e0880c](https://github.com/synonymdev/bitkit/pull/333/commits/4e0880cca3f3c647d76ce9496b2e170aee69311c)

**Comment by MiguelMedeiros** [comment]
> Can you mark in the Google Doc which tasks you have done? Maybe strikethrough or something like that so we know what is still open?

I'm opening up the issues that are still missing from the Aldert document.
So you can pick any open issue an assign to yourself. :+1:

File: src/screens/Wallets/AmountButtonRow.tsx [hunk 4]
```diff
@@ -93,14 +93,15 @@ const styles = StyleSheet.create({
...
+		paddingTop: 15,
 	},
 	topRowButtons: {
-		paddingVertical: 5,
+		paddingVertical: 6,
 		paddingHorizontal: 8,
 		borderRadius: 8,
 		flexDirection: 'row',
-		justifyContent: 'center',
+		justifyContent: 'space-between',
```
Comment (pwltr) [inline src/screens/Wallets/AmountButtonRow.tsx:110]: Not a fan of the spacing here.
Simulator Screen Shot - iPhone 13 - 2022-09-07 at 20 39 36

The design is not exact either here but I think it should look more like this:
Simulator Screen Shot - iPhone 13 - 2022-09-07 at 20 39 17

Suggestion - wrap each button in a container and do:
```suggestion
	buttonContainer: {
		alignItems: 'center',
		flex: 1,
	},
```

**Review by pwltr — CHANGES_REQUESTED** [review]

File: src/screens/Wallets/AmountButtonRow.tsx (hunk 4 above)
Comment (pwltr) [reply src/screens/Wallets/AmountButtonRow.tsx:110]: Really not sure what the design is saying here, maybe it was correct like you did it. Feel free to ignore..

Comment (MiguelMedeiros) [reply src/screens/Wallets/AmountButtonRow.tsx:110]: Fixed here: [baf022b](https://github.com/synonymdev/bitkit/pull/333/commits/baf022b597852309b00af9e53eea45f5e452a586)

**Review by pwltr — APPROVED** [review]

**Pull request merged**